### PR TITLE
fix: update AMI image with new key

### PR DESCRIPTION
**Pull request by fakirAyoub**
Update AMI image with valid token

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: build-and-deploy
 on:
+  push:
```
Comment (lumir-mrkva) [inline .github/workflows/tests.yml:3]: dont trigger workflow on push

**Review by lumir-mrkva — CHANGES_REQUESTED** [review]

**Comment by lumir-mrkva** [comment]
also the pr has to semantically named https://github.com/zeke/semantic-pull-requests

File: .github/workflows/tests.yml (hunk 1 above)
Comment (lumir-mrkva) [reply .github/workflows/tests.yml:3]: @fakirAyoub this is still an issue

Comment (fakirAyoub) [reply .github/workflows/tests.yml]: @lumir-mrkva change done

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**